pydantic_ai_slim/pydantic_ai/_cli/__init__.py

```diff
@@ -165,20 +239,41 @@ def cli(  # noqa: C901
             console.print(f'  {model}', highlight=False)
         return 0
 
+    if args.command == 'web':
+        from .web import run_web_command
+
+        return run_web_command(
+            agent_path=args.agent,
+            host=args.host,
+            port=args.port,
+            models=args.models,
+            tools=args.tools,
+            instructions=args.instructions,
+        )
+
+    # Default to chat command
```

Comment:
Is this the cleanest way to do this? I'm worried we'll forget to add some new arg to the default values list below. 

These stackoverflow posts may be relevant:
- https://stackoverflow.com/questions/46963172/how-do-you-get-argparse-to-choose-a-default-subparser
- https://stackoverflow.com/questions/46667843/how-to-set-a-default-subparser-using-argparse-module-with-python-2-7
- https://stackoverflow.com/questions/6365601/default-sub-command-or-handling-no-sub-command-with-argparse

(of course some of those "solutions" may be worse than this one)